src/cli.lisp

```diff
@@ -15,25 +15,72 @@
   (let ((raw (read-line)))
     (cl-ppcre:split "\\s+" raw)))
 
+(defparameter *pkey-hash* nil)
+(defparameter *pkey* nil)
+(defparameter *skey* nil)
+
+
 (defun main ()
   (let (terminate-cli)
+    (emotiq:set-owner "alice")
     (loop
-       :until terminate-cli
-       :doing (progn
-                (output-prompt)
-                (let ((input (read-command)))
-                  (cond
-                    ((string-equal (first input)
-                                   "quit")
-                     (setf terminate-cli t))
-                    ((string-equal (first input)
-                                   "create-wallet")
-                     (emotiq/wallet:create-wallet))
-                    ((string-equal (first input)
-                                   "help")
-                     (format *standard-output* "~&Available commands are~&~
+     :until terminate-cli
+     :doing (progn
+              (output-prompt)
+              (let ((input (read-command)))
+                (cond
+                 ((string-equal (first input)
+                                "quit")
+                  (setf terminate-cli t))
+                 ((string-equal (first input)
+                                "create-wallet")
+                  (emotiq/wallet:create-wallet))
+                 ((string-equal (first input)
+                                "open")
+                  (emotiq/wallet:open-wallet))
+                 ((string-equal (first input)
+                                "see")
+                  (emotiq/wallet::see-genesis))
+                 ((string-equal (first input)
+                                "test")
+                  (emotiq::test-blockchain))
+
+                 ((string-equal (first input)
```

Comment:
Please either remove the `test-*` invocations (probably preferred since they aren't part of the MVP as far as I know) or document their invocation in the help argument.